sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_v2/checks.sql

```diff
@@ -0,0 +1,54 @@
+#warn
+{{ is_unique(["client_id"], where="submission_date = @submission_date") }}
+
+#warn
+{{ min_row_count(1000, where="submission_date = @submission_date") }}
+
+#fail
+WITH daily AS (
+  SELECT
+    submission_date,
+    COUNT(DISTINCT client_id) AS client_count
+  FROM
+    `moz-fx-data-shared-prod.telemetry_derived.clients_daily_v6`
+  WHERE
+    submission_date = @submission_date
+  GROUP BY
+    submission_date
+),
+last_seen AS (
+  SELECT
+    submission_date,
+    COUNT(DISTINCT client_id) AS client_count
```

Comment:
Are we not potentially hiding problems here by doing count(DISTINCT client_id)? This table should only have a single client_id entry per submission_date, right?

Reply:
No because there is a separate check for duplicates. This check is specific to validate that all client_ids are included (completeness).